lightly/data/dataset.py

```diff
@@ -93,26 +93,50 @@ class LightlyDataset:
     """Provides a uniform data interface for the embedding models.
 
     Should be used for all models and functions in the lightly package.
-    Returns a tuple (sample, target, fname) when accessed using __getitem__
+    Returns a tuple (sample, target, fname) when accessed using __getitem__.
 
-    Can either be used to load a dataset offered by torchvision (e.g. cifar10)
-    or to load a custom dataset from an input folder.
+    The LightlyDataset supports different input sources. You can use it
+    on a folder of images. You can also use it on a folder with subfolders
+    with images (ImageNet style). If the input_dir has subfolders each subfolder
+    gets its own target label. You can also work with videos (requires pyav).
+    If there are multiple videos in the input_dir each video gets a different
+    target label assigned. If input_dir contains images and videos
+    only the videos are used.
+
+    Can also be used in combination with the `from_torch_dataset` method
+    to load a dataset offered by torchvision (e.g. cifar10).
 
     Args:
         input_dir:
-            Path to directory holding the images to load.
+            Path to directory holding the images or videos to load.
         transform:
             Image transforms (as in torchvision).
         index_to_filename:
             Function which takes the dataset and index as input and returns
             the filename of the file at the index. If None, uses default.
 
     Examples:
-        >>> # load cifar10 from a local folder
+        >>> # load a dataset consisting of images from a local folder
+        >>> # mydata
```

Comment:
Maybe add a slash here? `>>> # mydata/`
If you do it, also do it with the next three examples.